core/util/src/main/scala/net/liftweb/util/HtmlParser.scala

```diff
@@ -363,20 +363,20 @@ trait Html5Parser {
           if (capture) {
             val text = buffer.toString()
             if (text.length() > 0) {
-              hStack.push(createText(text))
+              hStack = createText(text) +: hStack
```

Comment:
In scala-xml 2.0.0 hStack has changed from `scala.collection.mutable.Stack` to `scala.collection.immutable.List`

https://github.com/scala/scala-xml/blob/main/CHANGELOG.md#200-2021-05-13